source/Octopus.Client.Tests/Serialization/MultiIsoDateTimeFormatConverterFixture.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using Octopus.Client.Serialization;
+
+namespace Octopus.Client.Tests.Serialization
+{
+    [TestFixture]
```

Comment:
A link back to https://github.com/OctopusDeploy/Issues/issues/5535 would be handy here to document why we have two dates.